server.js

```diff
@@ -0,0 +1,47 @@
+var port = process.env.PORT || 3000;
+
+// import modules.
+var path = require('path');
+var express = require('express');
+var app = express();
+var favicon = require('serve-favicon');
+var router = express.Router();
+
+// static middleware options.
+var staticOptions = {
+  dotFiles: 'ignore',
+  etag: true,
+  index: 'index.html',
+  lastModified: true,
+  redirect: true
+};
+
+// import routes.
+require('./routes')(router);
+
+// global middleware.
+app.use([express.static(path.join(__dirname, '/assets'), staticOptions), 
+  favicon(path.join(__dirname, '/assets/images/favicon.ico')),
+  router
+]);
+
+// 404 handler.
+app.use(function(req, res, next) {
```

Comment:
this 404 handler doesn't seem to be triggering for single-path resources.

i.e: `localhost:3000/something` brings me to a blank page, while `localhost:3000/something/else` takes me to the correct 404 page.